Dockerfile

```diff
@@ -5,9 +5,7 @@ WORKDIR /code
 
 # copy app
 COPY . .
-
-# install from local file
-RUN pip install .
+RUN pip install . && pip install -r requirements.txt
```

Comment:
Why? We don't want to install all testing and support requirements, thanks why we are installing from `setup.py`.

These packages aren't requred for runtime:
```
    'coveralls',
    'faker',
    'flake8',
    'pytest',
    'pytest-cov',
    'pytest-env',
    'pytest-flask',
    'pytest-ordering',
```

Reply:
BTW, `requirements.txt` isn't used anywhere. We keep it just be backward compatible and easier to onboard for new developers.

Reply:
ok, ill remove it